tests/test_feedexport.py

```diff
@@ -623,6 +653,23 @@ def test_export_no_items_store_empty(self):
             data = yield self.exported_no_data(settings)
             self.assertEqual(data[fmt], expctd)
 
+    @defer.inlineCallbacks
+    def test_export_no_items_multiple_feeds(self):
+        settings = {
+            'FEEDS': {
+                self._random_temp_filename(): {'format': 'json'},
+                self._random_temp_filename(): {'format': 'xml'},
+                self._random_temp_filename(): {'format': 'csv'},
+            },
+            'FEED_STORAGES': {'file': 'tests.test_feedexport.WriteOnStoreFileStorage'},
+            'FEED_STORE_EMPTY': False
+        }
+        data = yield self.exported_no_data(settings)
+
+        self.assertIn('json', data)
+        self.assertIn('xml', data)
+        self.assertIn('csv', data)
```

Comment:
I think this test could use a docstring. I think it is checking that `FEED_STORE_EMPTY=False` does not prevent `store()` from being called by using `WriteOnStoreFileStorage`, which acts as if `FEED_STORE_EMPTY=True`.

Also, based on your issue report, I wonder if it would not be easier to check the log content instead. I don’t see any log-based test in this test file, though, so maybe it would actually be harder.

Reply:
Actually, `storage.store` needs to be called every time by design.
`FEED_STORE_EMPTY` is a little bit strange setting. It does not prevent a file from being created, but rather it prevents creation of an object inside the file. 
In other words, this setting defines whether an output file will contain `[]` (in case of json) or will be empty.
This is a little confusing, in my opinion, but out of scope of this PR.

Here I fix the missing calls of `storage.store` that are expected even in cases when there are no data and `FEED_STORE_EMPTY=False`.